pkg/webhook/nodenetworkconfigurationpolicy/server.go

```diff
@@ -38,7 +21,6 @@ func Add(mgr manager.Manager) error {
 	// 2.- Since we have delete the condition the status-mutate webhook get called and
 	//     there we set conditions to Unknown this final result will be updated.
 	server := webhookserver.New(mgr, webhookName, certificate.MutatingWebhook,
-		webhookserver.WithPort(webhookPort),
```

Comment:
We still have targetPort in service. How does it work now without explicit port being specified?

Reply:
It will use kube-admission-webhook default port https://github.com/qinqon/kube-admission-webhook/blob/master/pkg/webhook/server/server.go#L35

Reply:
Then you gotta fix the manifest, `Service` still has `      targetPort: 54874`

Reply:
https://github.com/nmstate/kubernetes-nmstate/pull/497/files#diff-5e5f804a8ee2fb8c81711b728eb5a21dR150 still applies